gsa/src/gmp/models/result.js

```diff
@@ -49,6 +49,79 @@ export class Delta {
 class Result extends Model {
   static entityType = 'result';
 
+  static parseObject(object) {
+    const copy = super.parseObject(object);
+
+    const {
+      detectionResult,
+      host = {},
+      name,
+      notes,
+      nvt = {},
+      overrides,
+      originalSeverity,
+      qod = {},
+      severity,
+      task,
+      tickets,
+    } = object;
+
+    copy.host = {
+      name: host.ip,
+      id: isDefined(host.id) && !isEmpty(host.id) ? host.id : undefined,
```

Comment:
Should this be `hasValue`? I'm under the impression that hyperion backend always sends `null` and never `undefined`. `null` will return true with `isDefined` I think.

Reply:
isEmpty should return true for null

Reply:
yes currently i doesn't check the null case. it should be extend to support the null case.